scripts/qesap/test/unit/test_hy_conf.py

```diff
@@ -1,12 +1,14 @@
 import os
 import pytest
-from hypothesis import given
+from hypothesis import given, example
 from hypothesis.strategies import dictionaries, text
 from lib.config import CONF
 
 
 @pytest.mark.skipif("FUZZYTEST" not in os.environ, reason="Fuzzy test disabled by default")
-@given(dictionaries(text(), text()))
+@example({})
+@example({'apiver':'3','terraform':''})
+@given(dictionaries(text(), text()).filter(lambda x: 'ansible' not in x))
```

Comment:
Not strictly related to changes in this PR but was wrong and it result in PR workflow to fails